### DISCUSSION TITLE
How to use Queries ?

### DISCUSSION THREAD

**Discussion by ianquar** [strapi/strapi#6185] [Legacy Topics]
I want to use `strapi.query('restaurant').find({ id: 1 });`, but I don't know where do I put this in the `frontend` or `backend`, where exact?, how to call this function ?

**Comment by congthang1** [comment]
This backend, frontend use api endpoint or graphql

**Comment by robertpiosik** [comment]
This method is useful if you want to get the data within your backend application code.

In the browser/postman you should call `http://localhost:1337/restaurant/1` instead.
